src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory_Methods.vb

```diff
@@ -481,38 +477,100 @@ Namespace Microsoft.CodeAnalysis.Operations
         ''' Creates the Lazy IOperation from a delegate creation operand or a bound conversion operand, handling when the conversion
         ''' is actually a delegate creation.
         ''' </summary>
-        Private Function CreateConversionOperand(operand As BoundNode, conversionKind As ConversionKind, conversionSyntax As SyntaxNode, targetType As TypeSymbol
-                                                 ) As (Operation As Lazy(Of IOperation), IsDelegateCreation As Boolean)
-            If IsDelegateCreation(conversionKind, conversionSyntax, operand, targetType) Then
-                Dim operandLazy As Lazy(Of IOperation)
-                If operand.Kind = BoundKind.DelegateCreationExpression Then
-                    ' If the child is a BoundDelegateCreationExpression, we don't want to generate a nested IDelegateCreationExpression.
-                    ' So, the operand for the conversion will be the child of the BoundDelegateCreationExpression.
-                    ' We see this in this syntax: Dim x = New Action(AddressOf M2)
-                    ' This should be semantically equivalent to: Dim x = AddressOf M2
-                    ' However, if we didn't fix this up, we would have nested IDelegateCreationExpressions here for the former case.
-                    operandLazy = New Lazy(Of IOperation)(Function() CreateBoundDelegateCreationExpressionChildOperation(DirectCast(operand, BoundDelegateCreationExpression)))
-                Else
-                    ' This is either a lambda, or an AddressOf error scenario in which we have a delegate conversion, but it failed to bind correctly.
-                    ' Delegate to standard operation handling for this.
-                    operandLazy = New Lazy(Of IOperation)(Function() Create(operand))
+        Private Function GetConversionInfo(boundConversion As BoundConversionOrCast
+                                                 ) As (Operation As Lazy(Of IOperation), ConversionStruct As Conversion, IsDelegateCreation As Boolean)
```

Comment:
The alignment is strange, I would align the first character on this line with the first character on the previous line.